sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/FormRecognizerRecordedTestSanitizer.cs

```diff
@@ -16,6 +16,8 @@ public override void SanitizeHeaders(IDictionary<string, string[]> headers)
             {
                 headers[Constants.AuthorizationHeader] = new[] { SanitizeValue };
             }
+
+            base.SanitizeHeaders(headers);
```

Comment:
Calling `base` here since it sanitizes the `Authorization` header, which is being used for AAD auth.